library/src/main/java/org/apache/apex/malhar/lib/function/FunctionOperatorUtil.java

```diff
@@ -16,25 +16,28 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+package org.apache.apex.malhar.lib.function;
 
-package org.apache.apex.examples.transform;
+import java.io.PrintStream;
 
-import org.junit.Test;
-import org.apache.hadoop.conf.Configuration;
-import com.datatorrent.api.LocalMode;
-
-public class ApplicationTest
+/***
+ * A utility class that can be used in conjunction with the FunctionOperator class
+ */
+public class FunctionOperatorUtil
 {
-  @Test
-  public void testApplication() throws Exception
+
+  /***
+   * A Map Function implementation that can be used to write to the Console as a sink.
+   */
+  @SuppressWarnings("serial")
+  public static final Function.MapFunction<Object, Void> CONSOLE_SINK_FN = new Function.MapFunction<Object, Void>()
```

Comment:
change enricher example to also use this

Reply:
The Enricher Application also has been modified as part of this PR ( in the second iteration )